app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/tourism_information/AddInformationForm.kt

```diff
@@ -1,18 +1,33 @@
 package de.westnordost.streetcomplete.quests.tourism_information
 
 import android.os.Bundle
-import de.westnordost.streetcomplete.R
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.key
+import androidx.compose.ui.semantics.Role
 import de.westnordost.streetcomplete.quests.AImageListQuestForm
+import de.westnordost.streetcomplete.ui.common.image_select.ImageListItem
+import de.westnordost.streetcomplete.ui.common.image_select.SelectableIconCell
 
 class AddInformationForm : AImageListQuestForm<TourismInformation, TourismInformation>() {
 
     override val itemsPerRow = 2
 
     override val items = TourismInformation.entries.map { it.asItem() }
 
+    override val itemContent =
+        @Composable { item: ImageListItem<TourismInformation>, index: Int, onClick: () -> Unit, role: Role ->
+            key(item.item) {
```

Comment:
what does this `key(item.item)` do? Can't this be outside of this function, i.e. in `AImageListQuestForm.kt` line 118? (Or `ImageGrid.kt` line 32? I have no idea what it does, but it is certainly some boilerplate)

Reply:
it basically just tells compose, to recalculate the whole thing when that object changes, it is needed in the ones that can change on orientation or they don't change , I thought it was good practice to keep them elsewhere too